docs/reference/modules/cluster/allocation_filtering.asciidoc

```diff
@@ -50,9 +50,14 @@ PUT _cluster/settings
 The cluster allocation settings support the following built-in attributes:
 
 [horizontal]
-`_name`::   Match nodes by node names
-`_ip`::     Match nodes by IP addresses (the IP address associated with the hostname)
-`_host`::   Match nodes by hostnames
+`_name`::       Match nodes by node names
+`_host_ip`::    Match nodes by host IP address (IP associated with hostname)
+`_publish_ip`:: Match nodes by publish IP address
+`_ip`::         Match nodes by IP addresses (the IP address associated with the hostname)
```

Comment:
`Match nodes by IP addresses (the IP address associated with the hostname)` -> `Match either `_host_ip` or `_publish_ip`